packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -43,7 +43,7 @@ export const messages = {
   force_lightning_lwc_preview_unsupported:
     "Something's not right with the filepath. The local development server doesn't recognize the Lightning Web Components module '%s.'",
   force_lightning_lwc_open_text:
-    'SFDX: Open Local Development Server in Browser',
+    'SFDX: Open Code Preview Home',
```

Comment:
@ekapner shouldn't this be `SFDX: Open LWC Code Preview Home` ?

Reply:
@smaddox-sf - in the work item, we didn't say "LWC" - but should we? I'm trying to remember if we discussed it and opted not to include it. Thanks @lcampos for pointing out the inconsistency.